src/commands/connections.test.ts

```diff
@@ -124,9 +120,7 @@ describe("commands/connections.ts", function () {
 
   it("getSSLPemPaths() should return an empty array if the path is not set", function () {
     const nullConfig = {
-      get: sandbox.stub().callsFake((key: string, defaultValue?: unknown) => {
-        if (key === SSL_PEM_PATHS) return defaultValue;
-      }),
+      get: sandbox.stub().withArgs(SSL_PEM_PATHS, []).returns([]),
```

Comment:
I was debating removing this test altogether since we use a default empty array, but it's fine to keep in for now
https://github.com/confluentinc/vscode/blob/b3f8c70e863643855244d8d0176a839e0c246628/src/commands/connections.ts#L156-L158